src/search/SearchDrawer.js

```diff
@@ -24,12 +24,18 @@ const useStyles = makeStyles(styles, { name: 'RSFSearch' })
 export default function SearchDrawer({ DrawerComponent, classes, open, onClose, children }) {
   classes = useStyles({ classes })
 
+  const handleNavigation = () => {
+    if (onClose) {
```

Comment:
Since you've changed the propType to be required, this check isn't necessary.

Reply:
It actually is, otherwise navigator will crash and will prevent you from navigating. IsRequired will just throw warning that it is required.

Reply:
It won't crash if they obey the warning 😄

Reply:
True 😄